server/routerlicious/packages/lambdas/src/nexus/networkHelper.ts

```diff
@@ -19,23 +19,37 @@ export async function checkNetworkInformation(
 ): Promise<{ message: string; shouldConnect: boolean }> {
 	const tenantId = socket?.handshake?.query?.tenantId as string | undefined;
 	const tenantInfo = await tenantManager.getTenantfromRiddler(tenantId);
-	const privateLinkEnable = tenantInfo?.customData?.privateEndpoints?.accountLinkId
-		? true
-		: false;
+	const privateLinkEnable =
```

Comment:
Can this be simplified to something like:

const privateLinkEnable = !!tenantInfo?.customData?.privateEndpoints?.[0]
  ?.privateEndpointConnectionProxy?.properties?.remotePrivateEndpoint?.connectionDetails?.[0];

Reply:
Which basically says: Enable if the first privateEndpoint exists and has at least one connectionDetails entry.